mockwebserver/build.gradle

```diff
@@ -8,7 +8,7 @@ jar {
 
 dependencies {
   api project(':okhttp')
-  api deps.junit
+  compileOnly deps.junit
```

Comment:
I think I’d prefer we keep this as-is to preserve compatibility, and introduce a new module that has all the code

Reply:
I think this is actually cleaner.  You should only be affected after an upgrade of dependencies, and when your tests no longer run because you are missing junit the fix is trivial and obvious.  But critically, it avoids us having the confusion of an additional module.

Reply:
If this change popped up at runtime of the app, I'd 100% agree with you.  What module would you suggest if we split it out?